### ISSUE TITLE
[Bug] `QueryMessages` call only return one message

### ISSUE THREAD

**Issue by kuhnbv** [DKE-Data/agrirouter#40]
### Contact Details

dev_agrirouter@kuhn.com

### What happened?

In our application, we query the message headers of an Agrirouter endpoint to later, retrieve the corresponding messages and store them in our database.

The headers part seems to work fine, I get a list of the pending message headers. My program simply take all those messages Ids and make a `QueryMessages` call to Agrirouter.

This is were it seem to be malfunctionning, when I give this list of message ids, I only get at most one message even if I give more than one Id to retrieve. 

I check [your documentation](https://docs.agrirouter.com/agrirouter-interface-documentation/latest/commands/feed.html#message-query-and-filtering) and the C# SDK `QueryMessagesParameters`, both are talking about a list of Ids :

```csharp
namespace Agrirouter.Api.Service.Parameters
{
    /// <summary>
    ///     Parameter container definition.
    /// </summary>
    public class QueryMessagesParameters : MessageParameters
    {
        public List<string> Senders { get; set; }
        public List<string> MessageIds { get; set; } // 👈 here!
        public ValidityPeriod ValidityPeriod { get; set; }
    }
}
```

So as a temporary fix, I made a `foreach` loop to query one message at a time. But this introduce a lot of latency in our application and cache.

### agrirouter SDK

C#/.NET

### Support Data

```json
{
    "application_id": "5a3c2260-71c9-11ef-a06d-9a6c242172ce",
    "version_id": "4eb90e36-71ca-11ef-a06d-9a6c242172ce",
    "account_id": "27246708-eb09-4599-835d-7eaa2745ffce",
    "endpoint_id": "e3aeb095-18cc-4e4e-b240-a714c35bf779",
    "endpoint_external_id": "urn:uuid:e6a85db0-2cb2-451e-94b8-ab259b7a968d"
}
```

### Additional information

Our application use a router device.

**Comment by oliverrahner** [comment]
We are restricting the messages to be returned by both number of messages (max. 25 per request) as well as total size of the answer.
So if the first message you are requesting is quite large, what you see is to be expected.
Can you confirm that this might be the reason for what you are seeing?

**Comment by kuhnbv** [comment]
> We are restricting the messages to be returned by both number of messages (max. 25 per request) as well as total size of the answer. So if the first message you are requesting is quite large, what you see is to be expected. Can you confirm that this might be the reason for what you are seeing?

I wasn't aware of this restriction. When you say "quite large" what's the exact number? Where can I find more information about this restriction?

**Comment by oliverrahner** [comment]
The information can be found in the [developer docs](https://docs.agrirouter.com/agrirouter-interface-documentation/latest/commands/feed.html#paging). The information regarding the actual limits have just been added and couldn't be found before.
The main reason of this limitation originally was the restriction of the old agrirouter that couldn't process larger messages internally. The agrirouter 2.0 technically doesn't have this restriction, but during the switch we recognized that we needed to re-add this artificial limit because some partners' implementations relied on it.
